KM4K.py

```diff
@@ -56,15 +59,31 @@ def read_nfc():
                 tag.sys = 3
                 idm = binascii.hexlify(tag.idm)
                 return idm
-
+            
+def check_card_manager(idm):
+    url = "https://card.ueckoken.club/api/card/verify"
+    payload = json.dumps({
+      "idm": idm
+    })
+    headers = {
+      'x-api-key': os.environ["API_KEY"],
```

Comment:
[nits]
ヘッダのキーは次のように記載するルールがあったと記憶しています
`X-Api-Key`
```suggestion
      'X-Api-Key': os.environ["API_KEY"],
```